templates/course/lesson-preview.php

```diff
@@ -63,6 +63,31 @@
 				do_action( 'llms_lesson_preview_before_title', $lesson )
 				?>
 				<div class="llms-lesson-title"><?php echo esc_html( get_the_title( $lesson->get( 'id' ) ) ); ?></div>
+
+				<?php
+					$has_quiz = $lesson->is_quiz_enabled();
```

Comment:
Not a huge deal but could we put this inline within the `if` rather than having a temp variable `$has_quiz` that we don't use again?

Reply:
Completed.